### PR TITLE
feat[test]: Included a test case to verify the bulk upgrade of volumes

### LINKED ISSUE #268: Automate the test cases for Bulk Volume upgrade

### PR DESCRIPTION
**Pull request by nsathyaseelan**
Signed-off-by: nsathyaseelan <sathyaseelan.n@mayadata.io>

- Included a test case to upgrade the pools and volumes 
**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: providers/openebs/bulk-upgrade/cstor-spc-upgrade-job.j2 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+metadata:
+  name: cstor-spc-upgrade
+  namespace: {{ operator_ns }}
+spec:
+  backoffLimit: 4
+  template:
+    spec:
+      serviceAccountName: service_account
+      containers:
+      - name:  upgrade
```
Comment (gprasath) [inline providers/openebs/bulk-upgrade/cstor-spc-upgrade-job.j2]: remove double space

**Review by w3aman — APPROVED** [review]
/looks good

File: providers/openebs/bulk-upgrade/run_litmus_test.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          - name: logs
+            mountPath: /var/log/ansible 
+      volumes: 
+        - name: kubeconfig
+          configMap: 
+            name: kubeconfig 
+        - name: logs 
+          hostPath:
+            path: /mnt/openebs
+            type: ""
```
Comment (gprasath) [inline providers/openebs/bulk-upgrade/run_litmus_test.yml]: why do we need this volume?

File: providers/openebs/bulk-upgrade/test.yml [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+                  register: output
+                  until: "'installed' in output.stdout"
+                  delay: 30
+                  retries: 5
+
+                - name: Setting up tiller.
+                  shell: helm init
+                  args:
+                    executable: /bin/bash
+                  register: tiller_out
```
Comment (gprasath) [inline providers/openebs/bulk-upgrade/test.yml]: this may not be required if helm >3.0 is used

**Review by gprasath — APPROVED** [review]

**Pull request merged**